src/concat.js

```diff
@@ -19,15 +19,16 @@ var toString = require('./toString');
  * @category List
  * @sig [a] -> [a] -> [a]
  * @sig String -> String -> String
- * @param {Array|String} a
- * @param {Array|String} b
- * @return {Array|String}
+ * @param {Array|String} firstList The first list
+ * @param {Array|String} secondList The second list
```

Comment:
Question: Are there JSDoc tools that might choke if the parameter names in the document tag (`firstList`, `secondList`) don't match the ones in the actual function signature (`a`, `b`)?  Should we be concerned.